notebooks/2016-12-19-exploring_csw.ipynb

```diff
@@ -478,16 +478,21 @@
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "NOAA.NOS.CO-OPS Sensor Observation Service (SOS) Server\n",
-      "\n",
-      "This station provides the following variables: Air pressure, Air temperature, Sea surface height amplitude due to equilibrium ocean tide, Sea water temperature, Water surface height above reference datum, Wind from direction, Wind speed, Wind speed of gust\n"
+      "NOAA.NOS.CO-OPS Sensor Observation Service (SOS) Server  This station\n",
+      "provides the following variables: Air pressure, Air temperature, Sea\n",
+      "surface height amplitude due to equilibrium ocean tide, Sea water\n",
+      "temperature, Water surface height above reference datum, Wind from\n",
+      "direction, Wind speed, Wind speed of gust\n"
      ]
     }
    ],
    "source": [
+    "import textwrap\n",
+    "\n",
+    "\n",
     "value = csw.records['opendap.co-ops.nos.noaa.gov-urn_ioos_station_NOAA.NOS.CO-OPS_9439040']\n",
     "\n",
-    "print(value.abstract)"
+    "print('\\n'.join(textwrap.wrap(value.abstract)))"
```

Comment:
@jbosch-noaa I tested this in `Jekyll` and it wraps the output nicely in the HTML. However, I am not wrapping the ones with URLs because they became unreadable. I will let `Jekyll` deal with that by adding the horizontal scroll bar.